apps/web/components/chat/channel-summary-card.tsx

```
@@ -0,0 +1,191 @@
+    return (
+      <button
+        onClick={fetchSummary}
+        className="flex items-center gap-1.5 px-3 py-1 rounded-full text-xs font-medium transition-all motion-interactive motion-press"
+        style={{
+          background: "color-mix(in srgb, var(--theme-accent) 12%, transparent)",
+          color: "var(--theme-accent)",
+          border: "1px solid color-mix(in srgb, var(--theme-accent) 30%, transparent)",
+        }}
+        title="AI catch-up summary"
+        aria-label="Generate AI summary of recent messages"
+      >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)